### PR TITLE
Add "Traverso" AKA "Baroque Flute" to instruments

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
see https://musescore.org/en/node/85561

### CODE REVIEW

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -213,6 +213,24 @@
                         <program value="73"/>
                   </Channel>
             </Instrument>
+            <Instrument id="baroque-flute">
+                  <longName>Baroque Flute</longName>
+                  <shortName>Bq. Fl.</shortName>
+                  <description>Baroque Travers Flute</description>
```
Comment (lasconic) [inline share/instruments/instruments.xml]: is this really supposed to be "Travers" ?

Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: Is that Information used anywhere? I see it more like a comment. Or are you complaining about a missing e, "Traverse"?

Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: I'll fix that typo

**Comment by mgavioli** [comment]
Please see at https://musescore.org/en/node/85561#comment-379376 a few thoughts about why this addition might not be as simple as it may look at first sight.

**Pull request merged**